ts/screens/onboarding/PinScreen.tsx

```diff
@@ -150,7 +150,7 @@ class PinScreen extends React.Component<Props, State> {
           inactiveColor={variables.brandLightGray}
           activeColor={variables.brandDarkGray}
           onFulfill={this.onPinFulfill}
-          delayEnable={1000}
+          delayOnFailureMillis={1000}
```

Comment:
Actually this is the screen where the user provides the PIN the first time and confirms it, so I guess the "lock" behavior should be optional and disabled in this screen. You can make the `delayOnFailureMillis` optional, the lock/delay logic gets enabled only when `delayOnFailureMillis` is defined